src/test/coins_tests.cpp

```diff
@@ -751,21 +751,21 @@ BOOST_AUTO_TEST_CASE(ccoins_add)
     /* Check AddCoin behavior, requesting a new coin from a cache view,
      * writing a modification to the coin, and then checking the resulting
      * entry in the cache after the modification. Verify behavior with the
-     * with the AddCoin potential_overwrite argument set to false, and to true.
+     * with the AddCoin possible_overwrite argument set to false, and to true.
```

Comment:
nit: can remove the redundant "with the" (already present in the preceding line).

Reply:
done